thirdparty/libedit-3.1/examples/tc1

```diff
@@ -1,4 +1,4 @@
-#! /nix/store/mpgbs7f5x9k5cj5pa6hqk5rf7ph5hx6d-bash-4.4-p23/bin/bash
+#! /bin/sh
```

Comment:
`./configure` should already replace these paths with whatever makes sense for the particular system I think. If not, they should be generated by some other command in autotools

Also why is this file used in the first place?